docs/src/pages/styles/advanced/advanced.md

```diff
@@ -2,6 +2,8 @@
 
 <p class="description">This section covers more advanced usage of @material-ui/core/styles.</p>
 
+> **Note**: `@material-ui/styles` is the _**legacy**_ styling solution for Material-UI. It will be deprecated in v5. It depends on JSS as a styling solution, which is not used in the `@material-ui/core` anymore. If you would not like to have both emotion & JSS in your bundle, please refer to the [`@material-ui/system`](/system/basics/) documentation which is the recommended alternative.
```

Comment:
```suggestion
> **Note**: `@material-ui/styles` is the _**legacy**_ styling solution for Material-UI. It is deprecated in v5. It depends on [JSS](https://cssinjs.org/) as a styling solution, which is not used in the `@material-ui/core` anymore. If you don't want to have both emotion & JSS in your bundle, please refer to the [`@material-ui/system`](/system/basics/) documentation which is the recommended alternative.
```

x the other case

Reply:
added link to JSS in case some people don't know about it and I think "don't want" is easier to understand than "would not like".